services/ui-src/src/utils/reports/reports.test.ts

```diff
@@ -86,3 +90,40 @@ describe("Test createReportId", () => {
     expect(result).toEqual("AB_Program-Number-1_8-30-2022");
   });
 });
+
+describe("Sort Reports", () => {
+  it("should sort reports by oldest to newest", () => {
+    const unsortedReports = [
+      {
+        createdAt: 1662568556165,
+        programName: "Second Oldest",
+      },
+      {
+        createdAt: 1662568568589,
+        programName: "Third Oldest",
+      },
+      // Note this createdAt stamp is less than (older) than the prior 2 programs
```

Comment:
Nit: Don't necessarily need this comment as the program names should make it obvious. We could also add a readable time to them for future readers' sake, like:

```
{
    createdAt: 1662568556165, // 2022-09-07, 16:35:36 UTC
    programName: "Second Oldest",
},
```